resources/tests/GanacheTests/PrimOps.obs

```diff
@@ -0,0 +1,28 @@
+main contract PrimOps{
+        int x;
+        bool y;
+
+        transaction primops() {
+            //y = true and false; //these don't parse, or i have the syntax wrong
```

Comment:
Obsidian currently uses C-style names for these (&&, ||, !).

Reply:
ah! ok! i read the parser and thought i had it right, but thanks for the fix.